src/containers/NFT/NFTHeader/Details.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { CopyToClipboard } from '../../shared/components/CopyToClipboard'
+import './styles.scss'
+import { formatPrice } from '../../shared/utils'
+
+interface Props {
+  data: {
+    NFTId: string
+    ledgerIndex: number | undefined
+    owner: string
+    isBurned: boolean | undefined
+    flags: string[] | undefined
+    transferFee: number
+    issuer: string
+    NFTTaxon: number
+    NFTSequence: number | undefined
+    uri: string | undefined
+    validated: boolean | undefined
+    status: string | undefined
+    warnings: string[] | undefined
+    minted: string | undefined
+    domain: string | undefined
+    emailHash: string | undefined
```

Comment:
really, the shape of this should be divided. this looks like the intersection of an account and an nft, so it should be:

```typescript:
interface data = {
  NFT: NFT
  Account: Account
}

interface NFT = {
  ID: string
  ...
}

interface Account = {
  ID: string
  ...
}
```

then the `NFT` and `Account` definitions should be in another file that allows them to be referenced by lots of components and just imported here